src/main/java/au/org/ala/biocache/service/AlaImageMetadataService.java

```diff
@@ -54,23 +54,22 @@ public String getUrlFor(String imageId){
     }
 
     @Override
-    public Map<String, List<Map<String, Object>>> getImageMetadataForOccurrences(List<String> occurrenceIDs) throws Exception {
+    public Map<String, Map<String, Object>> getImageMetadataForOccurrences(List<String> imageIDs) throws Exception {
 
-        if (StringUtils.isBlank(imageServiceUrl) || occurrenceIDs == null || occurrenceIDs.isEmpty()){
-            return new HashMap<String, List<Map<String, Object>>>();
+        if (StringUtils.isBlank(imageServiceUrl) || imageIDs == null || imageIDs.isEmpty()){
+            return new HashMap<String, Map<String, Object>>();
```

Comment:
Shall we log here: 'The image service URL is not provided, return empty images , etc?'

Reply:
image-service is optional. It is acceptable to return no metadata when there is no metadata.